<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>teamList</string>
  </property>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>391</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Search By Team&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="east">
       <property name="text">
        <string>Eastern Conference</string>
       </property>
      </widget>
      <widget class="QPushButton" name="west">
       <property name="text">
        <string>Western Conference</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="teamList"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="teamListNext">
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToMainMenu">
    <property name="text">
     <string>Back to Main Menu</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
